src/issues/issue-2/david-byrne/Header.js

```diff
@@ -0,0 +1,35 @@
+import React from "react"
+import styled from "styled-components"
+
+import { TextBox, Text } from "./styled"
+import grit from "./grit"
```

Comment:
`grit` is a gradient string that adds a gritty effect to the element

Reply:
I discovered this effect in a SO article https://stackoverflow.com/questions/28183557/css-gradient-noise-as-div-background-filling